### ISSUE TITLE
how to use focal loss in ALbert model?

### ISSUE THREAD

**Issue by Thornhill-GYL** [HeyLynne/FocalLoss_for_multiclass#1]
Hi! can you share the code that you use focal loss in albert model? thanks!

**Comment by HeyLynne** [comment]
loss_fct = FocalLoss()
seq_loss = loss_fct(logits.permute(0, 2, 1), labels)
label_mask = attention_mask[:, 1:].float()
loss = ((seq_loss * label_mask).sum()) / label_mask.sum()

logits is the output of your model

**Comment by Thornhill-GYL** [comment]
Thanks for your replay! I used it in my model as you said,but there is error which says" seq_loss = loss_fct(logits.permute(0, 2, 1), labels) number of dims don't match in permute". Do you meet the problem before? and if you konw how to deal with it.That will help me a lot! Thanks

**Comment by HeyLynne** [comment]
You could print the logits and its shape.

**Comment by Thornhill-GYL** [comment]
OK 
I will figure it out by myself.
Thanks a lot

**Comment by HeyLynne** [comment]
Ok, hope you could figure it out smoothly.
